src/modules/units/components/UnitOverview.tsx

```diff
@@ -1,15 +1,30 @@
-import { Grid } from '@mui/material';
+import { Grid, Paper, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
 import React from 'react';
 
+import LoadingButton from '@/components/elements/LoadingButton';
 import MatchRuleCard from '@/modules/ce/components/unit/MatchRuleCard';
 import OpportunityBanner from '@/modules/ce/components/unit/OpportunityBanner';
-import { UnitDetailFieldsFragment } from '@/types/gqlTypes';
+import { cache } from '@/providers/apolloClient';
+import {
+  UnitDetailFieldsFragment,
+  useMarkUnitsAvailableMutation,
+} from '@/types/gqlTypes';
 
 interface Props {
   unit: UnitDetailFieldsFragment;
 }
 const UnitOverview: React.FC<Props> = ({ unit }) => {
   const opportunity = unit.latestOpportunity;
+  const [
+    markUnitAvailable,
+    { loading: availableLoading, error: availableError },
+  ] = useMarkUnitsAvailableMutation({
+    variables: { unitIds: [unit.id] },
+    onCompleted: () => cache.evict({ id: `Unit:${unit.id}` }),
```

Comment:
Nice to have: something like
```suggestion
    refetchQueries: ['GetUnit'],
    awaitRefetchQueries: true,
```
could help avoid the situation where the button becomes re-enabled for a second while the record is reloading.